templates/group-detail.html

```diff
@@ -1,20 +1,30 @@
 {% extends "base.html" %}
 {% import "macros.html" as macros %}
+
 {% block title %}Koschei - {{ group.full_name }} group{% endblock %}
+
+{% block subheader %}
+<h2>
+  {{ group.full_name }}
+  {% if group.editable %}
+  <a class="btn btn-secondary btn-sm" href="{{ url_for('edit_group', name=group.name, namespace=group.namespace) }}">
+    <i class="fa fa-pencil"></i>
+  </a>
+  {% endif %}
+</h2>
+<dl class="row">
+  <dt class="col-sm-3">Group maintainers:</dt>
+  <dd class="col-sm-9">
+    <ul class="list-inline">
+      {% for owner in group.owners %}
+      <li class="list-inline-item">{{ owner.name }}</li>
```

Comment:
Can you please join them with commas? Also the spacing looks a bit odd, maybe it would look better without columns

Reply:
Done.